src/views/forms/AccountManagementForms/SourceSelectMenu.tsx

```diff
@@ -14,43 +17,73 @@ import { getTransferInputs } from '@/state/inputsSelectors';
 
 type ElementProps = {
   label?: string;
+  selectedExchange?: string;
   selectedChain?: string;
-  onSelectChain: (chain: string) => void;
+  onSelectChain: (name: string, type: 'chain' | 'exchange') => void;
```

Comment:
should we rename this prop to reflect that it can be used for a chain or exchange? i.e. `onSelect`

Reply:
oh nice catch!